pixeltable/catalog/column.py

```diff
@@ -171,6 +171,33 @@ def get_idx_info(self) -> dict[str, 'TableVersion.IndexInfo']:
         assert self.tbl is not None
         return {name: info for name, info in self.tbl.get().idxs_by_name.items() if info.col == self}
 
+    @classmethod
+    def find_embedding_index(
```

Comment:
This is basically checking usage errors for the specific callers, which shouldn't live here. Move it  into ColumnRef.